code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridge.java

```diff
@@ -32,6 +32,10 @@
  */
 public class EdgeBridge {
 
+	private static final String CLASS_NAME = "EdgeBridge";
+
+	//	public static final Class<? extends Extension> EXTENSION = EdgeBridge.class;
```

Comment:
You will not need this here :) . See my comment on the extension class.

nit: also suggest removing commented code here and a few other places where it isn't blocking compilation.

Reply:
Sorry, I realized that my comment may have been unclear! I meant that the reference to `EdgeBridge.class` is not needed and that a reference to `EdgeBridgeExtension.class` is required.